src/commands/info/releasenotes/display.ts

```diff
@@ -31,64 +36,75 @@ export default class Display extends SfdxCommand {
 
   public static aliases = ['whatsnew'];
 
-  public static examples = messages.getMessage('examples', [Display.helpers.join(', ')]).split(os.EOL);
+  public static examples = messages.getMessage('examples', [Display.helpers.join('", "')]).split(os.EOL);
 
   protected static flagsConfig = {
     version: flags.string({
       char: 'v',
       description: messages.getMessage('flags.version'),
     }),
+    hook: flags.boolean({
+      hidden: true,
+      description: messages.getMessage('flags.hook'),
+    }),
   };
 
   public async run(): Promise<void> {
-    if (new Env().getBoolean('PLUGIN_INFO_HIDE_RELEASE_NOTES')) {
-      // We don't ever want to exit the process for info:releasenotes:display
+    const env = new Env();
+
+    if (env.getBoolean(HIDE_NOTES)) {
+      // We don't ever want to exit the process for info:releasenotes:display (whatsnew)
       // In most cases we will log a message, but here we only trace log in case someone using stdout of the update command
-      this.logger.trace('release notes disabled via env var: PLUGIN_INFO_HIDE_RELEASE_NOTES_ENV');
+      this.logger.trace(`release notes disabled via env var: ${HIDE_NOTES}`);
       this.logger.trace('exiting');
 
       return;
     }
 
-    const warn = (msg: string, err: Error): void => {
-      this.logger.trace(err.stack);
-      this.ux.warn(`${msg}\n${err.message}`);
-    };
-
-    const installedVersion = this.config.pjson.version;
-
-    let infoConfig: InfoConfig;
+    const isHook = !!this.flags.hook;
```

Comment:
probably unnecessary.  this.flags.hook` would give the same truthiness?

Reply:
Yea, that's correct. It'd either be `true` or `undefined`, both would work fine in an if statement (as long as you weren't using `??`). I chose `!!` because ensured a boolean and satisfied the ts compiler.